modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/notifications/NotificationsInfo.js

```diff
@@ -297,12 +297,11 @@ const NotificationsInfo = ({
 	}, [notificationIndex, recipientType, setSelectedItem]);
 
 	useEffect(() => {
-		if (recipientType === 'roleType') {
-			const sectionsData = [];
-
-			const recipients =
-				selectedItem.data.notifications.recipients[notificationIndex];
+		let sectionsData = [];
 
+		const recipients =
+		selectedItem.data.notifications && selectedItem.data.notifications.recipients[notificationIndex];
+		if (recipients && recipientType === 'roleType') {
```

Comment:
@MariMalta Please, add a blank line separating the definition and usage of recipients :)